### PR TITLE
Issue #762 added TRUNCATE to Local Storage Engine

### PR DESCRIPTION
**Pull request by seb-ster**
Added LS.truncateTable method to Local Storage Engine

### CODE REVIEW

File: test/test607.js [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+	var assert = require("assert");
+	var alasql = require('..');
+}
+
+var test = '607'; // insert test file number
+
+describe('Test '+test+' - TRUNCATE on Local Storage table', function() {
+
+	before(function(){
+		alasql('create LOCAL STORAGE database test'+test);
```
Comment (mathiasrw) [inline test/test607.js]: Please change `LOCAL STORAGE` to `LOCALSTORAGE`

**Comment by seb-ster** [comment]
Okay, @mathiasrw 
Sorry, i'm new at Github. I'm starting from scratch.
I'll have a new PR up soon.

**Comment by mathiasrw** [comment]
Sorry for a late reply!

Its all good - thats why we have the PR setup so everyon can get feedback on the code :)

The changes would need to be implemented in https://github.com/agershun/alasql/blob/develop/src/92localstorage.js

The folder is for a future code structure where every feature is a "plugin" to a core (so you only get the code you really need) - but its not implemented at all yet, so please ignore it...

**Comment by mathiasrw** [comment]
Hi @seb-ster 

I hope I did not scare you away with my inputs. 

Do you need any help to get further with the PR code?

**Comment by seb-ster** [comment]
No,  no problem. Just didn't get round to it.
Try to look into it today

**Comment by seb-ster** [comment]
So the commit has 13 files in it. Don't understand why. Only files i edited are 92localstorage.js and test607.js.
Please see the comments for test C and D as to why the would fail.

**Comment by mathiasrw** [comment]
Looks really good. 

Ill see if I can fix the merge things tomorrow.

**Comment by mathiasrw** [comment]
Finally got the merge to work. Thank you so much for your contribution :)

**Pull request merged**